lib/model.js

```diff
@@ -484,6 +484,7 @@ var validateIncludedElements = function(options, tableNames) {
         include.subQuery = false;
       } else {
         include.subQueryFilter = false;
+        include.subQuery = include.subQuery || include.hasParentRequired && (include.hasDuplicating || include.hasRequired);
```

Comment:
Ah you suggested to add `include.subQuery`, before adding that when I was looking for cause of the subQuery inclusion I found that `hasDuplicating` was also needed as deciding a factor. 

Now since logic has been changed, I'll need to check if we still need the `hasDuplicating` or any factor other than `include.subQuery`